tests/utils/shippable/shippable.sh

```diff
@@ -159,7 +196,7 @@ function cleanup
                         -f "${file}" \
                         -F "${flags}" \
                         -n "${test}" \
-                        -t 2b346748-05c5-499f-bb38-bd507bc0b6ef \
+                        -t 20636cf5-4d6a-4b9a-8d2d-6f22ebbaa752 \
```

Comment:
The `cleanup` function (and the trap) can be removed, as they're not used on AZP. They're just left over from when testing was done on Shippable.